src/components/cars/CarsCarousel.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import Paginate from './Paginate';
+import insta from '../../assets/images/insta-icon-home.png';
+import twitter from '../../assets/images/twitter-icon-home.png';
+import facebook from '../../assets/images/fb-icon-home.png';
+
+const social = [
+  { icon: facebook },
+  { icon: twitter },
+  { icon: insta },
+];
+
+const cars = [
+  { car: 'https://flyclipart.com/thumb2/aston-martin-cars-png-images-free-download-12690.png' },
+  { car: 'https://www.nicepng.com/png/detail/64-648130_southwest-custom-classic-cars-all-new-sport-cars.png' },
+  { car: 'https://png.pngtree.com/png-clipart/20210722/ourmid/pngtree-car-blue-transportation-reception-sports-car-png-image_6561592.png' },
+  { car: 'https://png.pngtree.com/png-clipart/20210722/ourmid/pngtree-car-red-trolley-sports-car-png-image_6561572.png' },
+];
+
+const CarsCarousel = () => {
+  const [pageNumber, setPageNumber] = useState(0);
+  const carsPerPage = 3;
+  const pageVisited = pageNumber * carsPerPage;
+  const displayCars = cars.slice(pageVisited, pageVisited + carsPerPage);
+  const pageCount = Math.ceil(cars.length / carsPerPage);
+
+  const changePage = ({ selected }) => {
+    setPageNumber(selected);
+  };
+
+  return (
+    <div className="flex flex-col gap-4 mt-16 lg:flex-row">
+      {displayCars.map((item) => (
+        <div className="flex flex-col items-center w-80 bg-green-200" key={item.name}>
```

Comment:
we should follow design strictly and bg collor green is not nessesery here.. single car should be wrapped with one circle and car should take background of thet div and with should be more than 100 %